AdminApi/src/AdminApi/AdminApi.csproj

```diff
@@ -8,74 +8,78 @@
+        <PackageReference Include="AspNetCore.HealthChecks.NpgSql" Version="8.0.1"/>
+        <PackageReference Include="AspNetCore.HealthChecks.SqlServer" Version="8.0.2"/>
     </ItemGroup>
     <ItemGroup>
-        <ProjectReference Include="..\..\..\BuildingBlocks\src\BuildingBlocks.API\BuildingBlocks.API.csproj" />
-        <ProjectReference Include="..\..\..\Modules\Challenges\src\Challenges.Application\Challenges.Application.csproj" />
-        <ProjectReference Include="..\..\..\Modules\Challenges\src\Challenges.Infrastructure\Challenges.Infrastructure.csproj" />
-        <ProjectReference Include="..\..\..\Modules\Devices\src\Devices.Application\Devices.Application.csproj" />
-        <ProjectReference Include="..\..\..\Modules\Devices\src\Devices.Infrastructure\Devices.Infrastructure.csproj" />
-        <ProjectReference Include="..\..\..\Infrastructure\Infrastructure.csproj" />
-        <ProjectReference Include="..\..\..\Modules\Quotas\src\Quotas.Application\Quotas.Application.csproj" />
-        <ProjectReference Include="..\..\..\Modules\Quotas\src\Quotas.Infrastructure\Quotas.Infrastructure.csproj" />
-        <ProjectReference Include="..\AdminApi.Infrastructure.Database.Postgres\AdminApi.Infrastructure.Database.Postgres.csproj" />
-        <ProjectReference Include="..\AdminApi.Infrastructure.Database.SqlServer\AdminApi.Infrastructure.Database.SqlServer.csproj" />
-        <ProjectReference Include="..\AdminApi.Infrastructure\AdminApi.Infrastructure.csproj" />
+        <ProjectReference Include="..\..\..\BuildingBlocks\src\BuildingBlocks.API\BuildingBlocks.API.csproj"/>
+        <ProjectReference Include="..\..\..\Modules\Challenges\src\Challenges.Application\Challenges.Application.csproj"/>
+        <ProjectReference Include="..\..\..\Modules\Challenges\src\Challenges.Infrastructure\Challenges.Infrastructure.csproj"/>
+        <ProjectReference Include="..\..\..\Modules\Devices\src\Devices.Application\Devices.Application.csproj"/>
+        <ProjectReference Include="..\..\..\Modules\Devices\src\Devices.Infrastructure\Devices.Infrastructure.csproj"/>
+        <ProjectReference Include="..\..\..\Infrastructure\Infrastructure.csproj"/>
+        <ProjectReference Include="..\..\..\Modules\Quotas\src\Quotas.Application\Quotas.Application.csproj"/>
+        <ProjectReference Include="..\..\..\Modules\Quotas\src\Quotas.Infrastructure\Quotas.Infrastructure.csproj"/>
+        <ProjectReference Include="..\AdminApi.Infrastructure.Database.Postgres\AdminApi.Infrastructure.Database.Postgres.csproj"/>
+        <ProjectReference Include="..\AdminApi.Infrastructure.Database.SqlServer\AdminApi.Infrastructure.Database.SqlServer.csproj"/>
+        <ProjectReference Include="..\AdminApi.Infrastructure\AdminApi.Infrastructure.csproj"/>
+
+        <Content Include="..\..\..\appsettings.override.json" Condition="$(Configuration) == Debug">
+            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
We don't want to make changes in the target dir, I think. So we can only copy it if it was updated.
Same for the other files

Reply:
I'm not sure what You mean with that, the documentation for the PreserveNewest property says:
> This item will only be copied if it is newer than the already existing item in the destination

Reply:
What I meant is that if you edit the file in the output directory (intentionally or not), it will not be replaced until you updated the original one.
But since that's rather an edge case, and I can't find a value that's a better fit, I'm also okay with leaving it as it is.